tests/test_report.py

```diff
@@ -390,6 +390,17 @@ def test_sorted_print(self):
         for expected, actual in zip(self.entries, pt_s):
             self.assertEqual(expected, actual)
 
+    def test_sorted_list_print(self):
+        """The internal PT should be sorted on the first column.
+        """
+        report = Report(self.headings, sort_by=[0, 1])
+
+        report.add_rows(self.out_of_order)
+        pt_s = get_report_printed_list(report)
+
+        for expected, actual in zip(self.entries, pt_s):
+            self.assertEqual(expected, actual)
+
```

Comment:
The test you added is a good one to show that asking it to sort with `0,1` sorts first on column 1 putting it in the order `[e3, e1, e2]`, and then sorts on column 0, which puts it in the order `[e1, e2, e3]`, but it doesn't test that it does a stable sort. This test would check that minimally:

```suggestion

    def test_sorted_list_with_matches_print(self):
        """Test sort on first and second column when elements are equal in first column
        """
        e1 = ['alice', 'mars', 'red']
        e2 = ['bob', 'venus', 'blue']
        e3 = ['charlie', 'earth', 'purple']
        e4 = ['alice', 'earth', 'green']
        out_of_order = [e1, e3, e2, e4]
        expected_order = [e4, e1, e2, e3]
        report = Report(self.headings, sort_by=[0, 1])
        report.add_rows(out_of_order)
        
        pt_s = get_report_printed_list(report)

        self.assertEqual(expected_order, pt_s)
```